app/models/effective/page.rb

```diff
@@ -30,7 +18,7 @@ class Page < ActiveRecord::Base
                           highlight: true,
                         }
                       },
-                      ranked_by: ":trigram" # Could rank by any column/expression, e.g.: (books.num_pages * :trigram) + (:tsearch / 2.0)
+                      if: -> (page) { !page.draft }
```

Comment:
doesn't matter here, but this could be page.published? to keep consistent with pages/events where it does matter. ty!